src/main/java/com/minecolonies/coremod/colony/requestsystem/requests/StandardRequests.java

```diff
@@ -275,7 +275,8 @@ public ITextComponent getShortDisplayString()
 
             if (posInList >= 0)
             {
-                return ImmutableList.of(new TranslationTextComponent(FROM, requester), new TranslationTextComponent(IN_QUEUE, posInList));
+            	if(posInList == 0) 	return ImmutableList.of(new TranslationTextComponent(FROM, requester), new TranslationTextComponent(IN_PROGRESS));
```

Comment:
Still inline it with ? :

Reply:
WDYM?

Reply:
return posInList == 0 ? ImmutableList.of() : ImmutableList.of()

Reply:
Oh `?:` have only slightly heard of it, and never used it. Gimme a sec

Reply:
Thank you for showing me this

Reply:
Np